tests/SideBySide.New/StoredProcedureTests.cs

```diff
@@ -0,0 +1,149 @@
+﻿using System;
+using System.Data;
+using System.Threading.Tasks;
+using MySql.Data.MySqlClient;
+using Xunit;
+
+namespace SideBySide
+{
+	public class StoredProcedureTests : IClassFixture<DatabaseFixture>, IDisposable
+	{
+		public StoredProcedureTests(DatabaseFixture database)
+		{
+			m_database = database;
+			m_database.Connection.Open();
+		}
+
+		public void Dispose()
+		{
+			m_database.Connection.Close();
+		}
+
+		[Fact]
+		public async Task StoredProcedureBasic()
+		{
+			using (var cmd = m_database.Connection.CreateCommand())
+			{
+				cmd.CommandText = @"DROP PROCEDURE IF EXISTS simple;
```

Comment:
It appears that when you use a `ReturnValue` parameter, Connector/Net assumes you are calling a function, not a stored procedure. The following code (inspired by https://dev.mysql.com/doc/connector-net/en/connector-net-tutorials-stored-procedures.html) works with Connector/Net:

```
			cmd.CommandText = "simple";
			cmd.CommandType = CommandType.StoredProcedure;
			using (var reader = await cmd.ExecuteReaderAsync())
			{
				Assert.True(await reader.ReadAsync());
				Assert.Equal("simple", reader.GetString(0));
			}
```